services/pipeline/src/main/java/org/hyperledger/besu/services/pipeline/Pipeline.java

```diff
@@ -174,15 +174,16 @@ private Future<?> runWithErrorHandling(final ExecutorService executorService, fi
             if (t instanceof CompletionException
                 || t instanceof CancellationException
                 || t instanceof AsyncOperationException) {
-              LOG.debug("Unhandled exception in pipeline. Aborting.", t);
+              LOG.trace("Unhandled exception in pipeline. Aborting.", t);
             } else {
-              LOG.info("Unexpected exception in pipeline. Aborting.", t);
+              LOG.info("Unexpected exception in pipeline. Aborting.");
+              LOG.debug("Unexpected exception in pipeline. Aborting.", t);
```

Comment:
nit: little annoying most of the log message except for the throwable will be logged twice once at info and also at trace level. But it's only a short message and at trace, you expect a lot of noise so it's probably fine.